### ISSUE TITLE
May I ask if the weights in the folder 'weights' have been trained? The results I obtained seem to be inaccurate, is it because I didn't train and used them directly?

### ISSUE THREAD

**Issue by jungyin** [KegangWangCCNU/PhysBench#16]
I tried using the weight file under weights to directly predict heart rate, but the results were not stable. So, I tried following the method in the readme and checked the RPPG signal using the test set. I found a huge difference between the RPPG signal and the label. Is this difference because the model I am using has not been trained?
![image](https://github.com/user-attachments/assets/2b02f687-32fb-49f4-adf9-132fe6e8771c)
![image](https://github.com/user-attachments/assets/95f6af97-e356-40ec-b74d-39b005a23ea9)
![image](https://github.com/user-attachments/assets/f441ef4a-d523-416c-8190-81421c0e83ef)
![image](https://github.com/user-attachments/assets/48194e8b-0b64-4f93-82f8-66099af7f252)

**Comment by KegangWangCCNU** [comment]
I need some details to determine whether you have correctly loaded the pre-trained weights. If possible, you could try testing on the PURE or UBFC-rPPG datasets, as the model usually performs well on these datasets. If you would like more suggestions, you could post your code for loading the weights and conducting the tests.
